### ISSUE TITLE
Improved 'fonts' task

### ISSUE THREAD

**Issue by hindsholm** [Nykredit/gript#39]
The `fonts` task scans `bower_components` and includes every font found in the build.
This is not an optimal solution for two reasons:

- Scanning all of `bower_components` is slow
- You might get more fonts than you want, e.g. the `modernizr` package includes some fonts for testing that you probably don't want even though you use `modernizr`

I have used this task as a workaround for a specific project:
```
gulp.task('fonts', function () {
    return gulp.src('bower_components/material-design-icons/iconfont/**/*.woff*')
        .pipe(gulp.dest('target/tmp/fonts'))
        .pipe(gulp.dest('target/dist/fonts'));
});
```
but a more permanent solution would be welcome :)

I guess you should be able to configure which fonts to include.

**Comment by jotka** [comment]
Well, we will need of some kind of a list (in config section of the `gulpfile.js`), saying what fonts we want to include.
Scanning the whole `node_modules` will remain as a default behavior.

Will such solution work for you ?

**Comment by radzikp** [comment]
We would prefer to set a list of folders to scan by gulp. It could be `gulpfile.js`.

**Comment by hindsholm** [comment]
Yes, a list of folders in the gulpfile configuration would be fine for me.

**Comment by jotka** [comment]
ok then, expect the update :sunglasses:

**Comment by jotka** [comment]
released 0.0.37
